.editorconfig

```diff
@@ -0,0 +1,12 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+

```

Comment:
are you sure this file look like this? maybe consult with Mariya?

Reply:
fixed